apps/web/src/studio/components/GetStartedPageV2/index.tsx

```diff
@@ -199,7 +203,16 @@ export const GetStartedPageV2 = () => {
                   marginBottom: '150',
                 })}
               >
-                <IconFolderOpen size={16} /> <span>Discover examples</span>
+                <IconFolderOpen
+                  size={16}
+                  className={css({
+                    color: {
+                      _dark: 'legacy.BGLight !important',
+                      base: 'legacy.B20 !important',
+                    },
+                  })}
+                />{' '}
+                <span>Discover examples</span>
```

Comment:
Is there an hover state for the buttons? like, a color change?

Reply:
No